pkg/interface/src/views/apps/chat/components/ChatWindow.tsx

```diff
@@ -317,7 +323,7 @@ class ChatWindow extends Component<
           onScroll={this.onScroll}
           data={graph}
           size={graph.size}
-          pendingSize={pendingSize}
+          pendingSize={pendingSize + contactsModified}
```

Comment:
Why are we adding 100 if we hide our contact?

Reply:
To force a re-render of the virtual scroller